ui/src/app/views/workspace-edit/component.html

```diff
@@ -15,140 +15,167 @@ <h2 *ngSwitchCase="Mode.Clone">Clone workspace "{{oldWorkspaceName}}"</h2>
         <div>National Institute of Health:</div>
       </span>
     </div>
-
     <div>
-      <h3>Research Purpose</h3>
-      <!-- TODO: Real description of workspace purpose -->
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. Proin sodales pulvinar tempor. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
       <div class="card">
         <h4>Describe your research purpose:</h4>
         <textarea class="input description" name="workspace-description"
             [(ngModel)]="workspace.description" [disabled]="!hasPermission">
         </textarea>
       </div>
       <div class="card">
-        <h4>Please select all that apply</h4>
+        <h4>Please select all data use categories that apply for your current study</h4>
         <div class="form-group">
           <div class="checkbox-group">
             <clr-checkbox
                 name="disease-focused-research"
                 id="disease-focused-research"
                 [(ngModel)]="workspace.researchPurpose.diseaseFocusedResearch"
                 [clrDisabled]="mode == Mode.Edit">
-              Disease focused research
+              <h3>Disease focused research</h3>
             </clr-checkbox>
-            <input type="text" class="input" name="disease-specification"
-                [(ngModel)]="workspace.researchPurpose.diseaseOfFocus" [disabled]="mode == Mode.Edit"
-                placeholder="Name of Disease">
-            <span style="display: inline-block; vertical-align: bottom; padding-left: 1rem; width: 25rem">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo.
-            </span>
+            <div style="display: flex">
+              <input type="text" class="input" name="disease-specification"
+                  [(ngModel)]="workspace.researchPurpose.diseaseOfFocus" [disabled]="mode == Mode.Edit"
+                  placeholder="Name of Disease">
+              <span style="vertical-align: bottom; padding-left: 1rem; width: 18rem">
+                The primary purpose of the research is to learn more about a particular disease or disorder
+                (e.g., type 2 diabetes), a trait (e.g., blood pressure), or a set of related conditions
```

Comment:
nit: on word choice, I've generally followed guidance to avoid e.g./i.e. in user-facing text, as it's not as common outside of technical / academic circles. I generally replace e.g. with "for example".